pixi.toml

```
@@ -17,7 +17,7 @@ python = "3.12.*"
 [tasks]
 build = "cargo build --release"
 bump = "tbump --only-patch $RELEASE_VERSION"
-install = "cargo install --path . --locked"
+install = { cmd = "python scripts/install.py", depends-on = ["build"] }
```

```suggestion
install = "cargo install --path . --locked"
install-as = { cmd = "python scripts/install.py", depends-on = ["build"] }
```

And then make the `--name` a positional so you can run `pixi run install-as pixi-bla`

---

Thanks!